MrtCore/winui_desktop_packaged_app/winui_desktop_packaged_app.sln

```diff
@@ -18,6 +18,8 @@ Global
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
 		{28B258CA-5ECC-41EE-A5B0-C2F24F3A4984}.Debug|Any CPU.ActiveCfg = Debug|x86
+		{28B258CA-5ECC-41EE-A5B0-C2F24F3A4984}.Debug|Any CPU.Build.0 = Debug|x86

```

Comment:
>x86 [](http://example.com/codeflow?start=71&length=3)

any cpu?

Reply:
this was auto generated by Visual Studio